### PR TITLE
Added a note about DualSense support on Android devices

### PR DESCRIPTION
**Pull request by JordanL8**
### Description

Adds information about DualSense support for Android devices.

### Changes made

Updated the SupportedDevices and changelog pages.

### Checklist

Before review:

- [x] Changelog entry added.

During merge:

- [ ] Commit message for squash-merge is prefixed with - `DOCS: ___`.

### CODE REVIEW

File: Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md [hunk 1]
```diff
@@ -42,10 +42,11 @@ Support for the following Devices doesn't require specialized support of particu
...
->9. Switch Joy-Cons are not currently supported on Windows and Mac. Some of official accessories are supported on Windows and Mac: "Hori Co HORIPAD for Nintendo Switch", "HORI Pokken Tournament DX Pro Pad", "HORI Wireless Switch Pad", "HORI Real Arcade Pro V Hayabusa in Switch Mode", "PowerA NSW Fusion Wired FightPad", "PowerA NSW Fusion Pro Controller (USB only)", "PDP Wired Fight Pad Pro: Mario", "PDP Faceoff Wired Pro Controller for Nintendo Switch", "PDP Faceoff Deluxe Wired Pro Controller for Nintendo Switch", "PDP Afterglow Wireless Switch Controller", "PDP Rockcandy Wired Controller".
->10. PS5 DualSense is supported on Windows and macOS via USB HID, though setting motor rumble and lightbar color when connected over Bluetooth is currently not supported.
->11. SteelSeries Nimbus+ supported via HID on macOS.
+>8. Unity supports PS4 controllers on Android devices running [Android 10 or higher](https://playstation.com/en-us/support/hardware/ps4-pair-dualshock-4-wireless-with-sony-xperia-and-android).
+>9. Unity supports PS5 controllers on Android devices running [Android 12 or higher](https://www.playstation.com/en-gb/support/hardware/pair-dualsense-controller-bluetooth/).
```
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md:46]: @JordanL8 thanks for making these changes. Is the Android support called out here relating to recent changes to Unity, if so what versions is this expected in or is it valid for all currently supported versions or only selected versions of the editor?

**Review by ekcoh — APPROVED** [review]

File: Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md (hunk 1 above)
Comment (JordanL8) [reply Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md:46]: I double-checked with the developer and it's not a recent change. Was just run into recently while testing

**Review by stefanunity — APPROVED** [review]
DualSense shows correct button mappings in remote Input debugger when connected via BT to Android 13 device (Pixel 6a). It reports as "Dualshock4GamepadAndroid" though. 
That is not stopping this change but something for the mobile team to follow up on.

**Comment by JordanL8** [comment]
Great, thanks. I'll let the mobile team know about that issue. With that, can someone merge this PR, please? 🙂

**Comment by ekcoh** [comment]
Rerouting this via: https://github.com/Unity-Technologies/InputSystem/pull/1622 to get CI runs, will close this PR when that PR is merged.

**Comment by ekcoh** [comment]
Rerouted PR has been merge so closing this one since now redundant

**Pull request closed without merging**